lakefs_provider/hooks/lakefs_hook.py

```diff
@@ -122,54 +110,53 @@ def get_branch_commit_id(self, repo: str, name: str) -> str:
 
     def get_commit(self, repo: str, ref: str) -> Dict[str, str]:
         client = self.get_conn()
-        # Actually ask for a log of length 1, because of treeverse/lakeFS#3436.
-        response = client.refs_api.log_commits(repo, ref, amount=1)
-        details = response.results[0]
-        return _commitAsDict(details)
+        commit = client.commits_api.get_commit(repo, ref)
+        return commit.to_dict()
```

Comment:
Nice!  This new?

Reply:
yea, some good things found in the new generated code.